src/styles/Components/src/tokens-list.css

```diff
@@ -0,0 +1,35 @@
+:root {
+  --denhaag-list-font-family: var(--denhaag-font-family);
+
+  /* ListItem */
+  --denhaag-listitem-padding-inline: 16px;
+  --denhaag-listitem-padding-block: 12px;
+  --denhaag-listitem-border-size: 2px;
+  --denhaag-listitem-focus-border-color: var(--denhaag-ocher-4);
+  --denhaag-listitem-focus-border-width: var(--denhaag-focus-border-width);
+  --denhaag-listitem-focus-border-style: var(--denhaag-focus-border-style);
+  --denhaag-listitem-padding-block-secondary: 8px;
+
+  /* ListItemIcon */
+  --denhaag-listitem-icon-color: var(--denhaag-grey-4);
+
+  /* ListItemSecondaryAction */
+  --denhaag-listitem-secondaryaction-focus-color: var(--denhaag-blue-3);
+  --denhaag-listitem-secondaryaction-color: var(--denhaag-grey-4);
+  --denhaag-listitem-secondaryaction-right: 16px;
+  --denhaag-listitem-secondaryaction-padding: 4px;
```

Comment:
I think: `--denhaag-list-item-secondary-action` would be better

Reply:
Agreed, we should always use `-` in between words where in regular language there would be a space. When in doubt use this rule: when a property/component name in MUI has a capital letter in the middle of its name this indicates the start of a new word, meaning we should insert a `-` in that position in the design token naming.

Reply:
I applied the - naming convention to all design tokens.